java-checks/src/test/java/org/sonar/java/checks/ImmediatelyReturnedVariableCheckTest.java

```diff
@@ -22,14 +22,16 @@
 import org.junit.jupiter.api.Test;
 import org.sonar.java.checks.verifier.CheckVerifier;
 import org.sonar.java.checks.verifier.TestUtils;
+import org.sonar.java.checks.verifier.internal.InternalCheckVerifier;
 
 class ImmediatelyReturnedVariableCheckTest {
 
   @Test
   void test() {
-    CheckVerifier.newVerifier()
+    ((InternalCheckVerifier) CheckVerifier.newVerifier())
```

Comment:
You don't need to cast. instead of 
```
((InternalCheckVerifier) CheckVerifier.newVerifier())
```
You can call directly
```
InternalCheckVerifier.newInstance()
```